subsys/mpsl/cx/CMakeLists.txt

```diff
@@ -4,11 +4,10 @@
 # SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
 #
 
-if (CONFIG_MPSL_CX_THREAD OR CONFIG_MPSL_CX_BT_1WIRE OR CONFIG_MPSL_CX_BT_3WIRE OR CONFIG_MPSL_CX_NRF700X)
+if (CONFIG_MPSL_CX_THREAD OR CONFIG_MPSL_CX_BT_1WIRE  OR CONFIG_MPSL_CX_NRF700X)
```

Comment:
Does it make sense to also remove the Kconfig option `MPSL_CX_BT_3WIRE` from `mpsl/cx/Kconfig`?